store/storethehash/storethehash_test.go

```diff
@@ -74,8 +76,19 @@ func TestPeriodicFlush(t *testing.T) {
 	}
 
 	value := indexer.MakeValue(p, 0, []byte(mhs[0]))
-	for _, c := range mhs[1:] {
-		_, err = s.Put(c, value)
+
+	// Check that a non-v0 CID hash can be stored.
+	c, err := cid.Decode("baguqeeqqskyz3yh4jxnsdj57v5blazexyy")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = s.Put(c.Hash(), value)
```

Comment:
validate you can get this cid back again as well?

Reply:
Done